tutorials/clinical/30_ecog.py

```diff
@@ -100,15 +100,11 @@
 # at the posterior commissure)
 raw.set_montage(montage)
 
-# Find the annotated events
-events, event_id = mne.events_from_annotations(raw)
-
 # Make a 25 second epoch that spans before and after the seizure onset
 epoch_length = 25  # seconds
 epochs = mne.Epochs(
     raw,
-    events,
-    event_id=event_id["onset"],
+    event_id=["onset"],
```

Comment:
This would work too, right?
```suggestion
    event_id="onset",
```

Reply:
Yes, similar to `event_id=1` for use with `events`

Reply:
Cool, shouldn't we pass the string, then? :) looks easier